appveyor.yml

```diff
@@ -1,5 +1,5 @@
 init:
-  - git config --global core.autocrlf true
+  - git config --global core.autocrlf input
```

Comment:
Okay, the autofix built into SourceLink only bumps the build time from 19 minutes to 20 minutes on AppVeyor. Not bad and may be acceptable instead of dealing with this build configuration stuff.